Puc/v4p2/Vcs/GitLabApi.php

```diff
@@ -0,0 +1,274 @@
+<?php
+
+if ( ! class_exists( 'Puc_v4p2_Vcs_GitLabApi', false ) ) :
+
+	class Puc_v4p2_Vcs_GitLabApi extends Puc_v4p2_Vcs_Api {
+		/**
+		 * @var string GitLab username.
+		 */
+		protected $userName;
+
+		/**
+		 * @var string GitLab server host.
+		 */
+		private $repositoryHost;
+
+		/**
+		 * @var string GitLab repository name.
+		 */
+		protected $repositoryName;
+
+		/**
+		 * @var string Either a fully qualified repository URL, or just "user/repo-name".
+		 */
+		protected $repositoryUrl;
```

Comment:
This property is already declared in the parent class and this subclass doesn't read or write it. As far as I can tell, there's no need to re-declare it here; please remove it.